src/ApparentHorizons/StrahlkorperGr.cpp

```diff
@@ -853,6 +855,39 @@ double christodoulou_mass(const double dimensionful_spin_magnitude,
   return sqrt(square(irreducible_mass) + (square(dimensionful_spin_magnitude) /
                                           (4.0 * square(irreducible_mass))));
 }
+
+template <typename Frame>
+void radial_distance(const gsl::not_null<Scalar<DataVector>*> radial_distance,
+                     const Strahlkorper<Frame>& strahlkorper_a,
+                     const Strahlkorper<Frame>& strahlkorper_b) {
+  if (strahlkorper_a.center() != strahlkorper_b.center()) {
+    ERROR(
+        "Currently computing the radial distance between two Strahlkorpers "
+        "is only supported if they have the same centers, but the "
+        "strahlkorpers provided have centers "
+        << strahlkorper_a.center() << " and " << strahlkorper_b.center());
+  }
+  get(*radial_distance)
+      .destructive_resize(strahlkorper_a.ylm_spherepack().physical_size());
+  if (strahlkorper_a.l_max() == strahlkorper_b.l_max() and
+      strahlkorper_a.m_max() == strahlkorper_b.m_max()) {
+    get(*radial_distance) =
+        abs(get(StrahlkorperFunctions::radius(strahlkorper_a)) -
+            get(StrahlkorperFunctions::radius(strahlkorper_b)));
+  } else if (strahlkorper_a.l_max() > strahlkorper_b.l_max() or
+             (strahlkorper_a.l_max() == strahlkorper_b.l_max() and
+              strahlkorper_a.m_max() > strahlkorper_b.m_max())) {
+    get(*radial_distance) = abs(
+        get(StrahlkorperFunctions::radius(strahlkorper_a)) -
+        get(StrahlkorperFunctions::radius(Strahlkorper<Frame>(
+            strahlkorper_a.l_max(), strahlkorper_a.m_max(), strahlkorper_b))));
+  } else {
+    get(*radial_distance) = abs(
```

Comment:
Do we want `radial_distance` to take the `abs`?  I can imagine cases where we want to know which strahlkorper is larger, and I can also imagine cases where we want to do some root-finding or control system to make two strahlkorpers agree with each other, and in both cases we would want `radial_distance` to have a sign.

Reply:
OK, the distance function no longer takes abs()